lighthouse-core/audits/byte-efficiency/uses-response-compression.js

```diff
@@ -21,7 +21,7 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
    */
   static get meta() {
     return {
-      name: 'uses-request-compression',
+      name: 'uses-response-compression',
```

Comment:
This is a better name, but should existing audits change names without a major version bump?

I'm admittedly completely biased because it's going to make my HTTP Archive queries (which are partly based on `runner.getAuditList()`) harder :)

Reply:
Yeah, that's fair, I was hoping it was new enough people weren't necessarily relying on it yet :/

Reply:
reverted